tests/e2e/testsLibrary/CodeExecutionTests.ts

```diff
@@ -21,7 +22,10 @@ const driverHelper: DriverHelper = e2eContainer.get(CLASSES.DriverHelper);
 export function runTask(taskName: string, timeout: number) {
     test(`Run command '${taskName}'`, async () => {
         await topMenu.runTask(taskName);
-        await ide.waitNotification('has exited with code 0.', timeout);
+        const taskExitCode : boolean = await ide.waitTaskExitCodeNotificationBoolean('0', timeout);
```

Comment:
Is it safe to test just `0`? Could it be triggered by some another notification? What about having larger text here?